docs/kubernetes/KubernetesPolicy.md

```diff
@@ -0,0 +1,63 @@
+# Programming Calico Policy in Kubernetes
+The [Calico Kubernetes plugin](https://github.com/projectcalico/calico-docker/blob/master/docs/kubernetes/KubernetesIntegration.md) allows you to specify networking policy in the Kubernetes API using pod annotations. 
+
+## Prerequisites
+* A Kubernetes Deployment
+    - To implement service policy using recent builds of Kubernetes, make sure you have configured your kube-proxy using the `--legacy-userspace-proxy=false` option
```

Comment:
Stick punctuation at the end of these guys.

I'd also mention that the v1.0 release of Kubernetes doesn't include the correct kube-proxy, and that you'll need to build Kubernetes master commit #abcdef or later to get the required kube-proxy behavior.